cmd/minishift/cmd/oc_env.go

```diff
@@ -32,31 +32,28 @@ import (
 )
 
 const (
-	ocEnvTmpl = `{{ .Prefix }}PATH{{ .Delimiter }}{{ .OcPath }}{{ .Suffix }}{{ .UsageHint }}`
+	ocEnvTmpl = `{{ .Prefix }}PATH{{ .Delimiter }}{{ .OcDirPath }}{{ .Suffix }}{{ .UsageHint }}`
 )
 
 type OcShellConfig struct {
 	shell.ShellConfig
-	OcPath    string
+	OcDirPath string
 	UsageHint string
 }
 
-func getOcShellConfig(ocPath string, forcedShell string) (*OcShellConfig, error) {
+func getOcShellConfig(ocPath, forcedShell string) (*OcShellConfig, error) {
 	userShell, err := shell.GetShell(forcedShell)
 	if err != nil {
 		return nil, err
 	}
 
 	cmdLine := "minishift oc-env"
 	shellCfg := &OcShellConfig{
```

Comment:
when ocPath is empty, fail the command...

this can happen when `"OcPath": "",` , because an install of `oc cluster up` didn't finish or didn't succeed.